contentcuration/contentcuration/tests/test_sync.py

```diff
@@ -429,6 +431,251 @@ def test_sync_channel_other_metadata_labels(self):
         for key, value in labels.items():
             self.assertEqual(getattr(target_child, key), {value: True})
 
+    def test_sync_language_field(self):
+        """
+        Test that the language field is synced correctly when sync_resource_details is True.
+        """
+        self.assertFalse(self.channel.has_changes())
+        self.assertFalse(self.derivative_channel.has_changes())
+
+        contentnode = (
+            self.channel.main_tree.get_descendants()
+            .exclude(kind_id=content_kinds.TOPIC)
+            .first()
+        )
+
+        target_child = self.derivative_channel.main_tree.get_descendants().get(
+            source_node_id=contentnode.node_id
+        )
+
+        self.assertIsNotNone(target_child)
```

Comment:
Dead code-- Django's `get()` should throw `ContentNode.DoesNotExist` if not found. This is repeated in each test.